README.md

```diff
@@ -1 +1,40 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+Project name : Hello Microverse
+Tech Stack:
+    Client: html, css
+    Server: none
+    Database: none
+Key feature:
+    - .gitignore
+    - linters
+Live Demo:
+    - link: in future
+
+Usage: free style
+Prequesites: none
+Deployment: none
+
+AUTHORS:
+    1. Khanh (Mr) 
+        - Github: khanh-devos, 
+        - Twitter: khanhror
+        - LinkedIn: Khanh Than Trong
+
+Contribution:
+    - All contributions are welcome
+
+Supports:
+    - If you like this project, can invite me a cup of tea
+
+Acknowlegments:
+    I would like to thank to everybody at Microverse
+
+License:
+    This project is MIT licensed.
+
+
+
+
+
```

Comment:
- [ ] For the project requirements, kindly ensure that your README file is descriptive and professional. And the required template to be used here is the **[Microverse README template](https://github.com/microverseinc/readme-template)**, so ensure that: 
  * All main headers from the [README template](https://github.com/microverseinc/readme-template) are present in a your repo.
  * The only optional section is `FAQ`
  *  you delete the `live demo` section if the you decide not to deploy the application.
  * Make sure the you have added the section `Future Features`, including 1-3 potential features to be added.
  * you remove the section `description`, i.e.` "> Describe the tech stack and include only the relevant sections that apply to your project."` should be removed and filled in with their custom content.
  * you remove template logo. Optionally, the you can replace it with your own logo you they have one.

Reply:
Thanks for your comment, I will improve it then commit again.

Reply:
Hello @tresorsawasawa 
I modified README file already.
But now I cannot change branch name because I try to keep the same request.

Reply:
Hi, @khanh-devos, don't worry about the branch name in this Pull Request, you can't change it because that'll modify the GitHub history, just take the recommendation for your future projects. 💯